Backend/model/Flight.js

```diff
@@ -27,14 +44,12 @@ const Flight = db.sequelize.define("Flight", {
   igcUrl: {
     type: DataTypes.STRING,
   },
+  imagesUrls: {
+    type: DataTypes.ARRAY(DataTypes.STRING),
+  },
   glider: {
     type: DataTypes.STRING,
```

Comment:
Same here maybe a glider object consisting of a UUID and glider name would be better?

Reply:
s.o.